roles/openshift_node/tasks/create_machine.yml

```diff
@@ -9,7 +9,7 @@
 
 - name: Check machine is provisioned
   shell: >
-    oc get machines --kubeconfig={{ kubeconfig }} -n openshift-machine-api |grep p3.2xlarge |awk '{ print $2 }'
+    oc get machines --kubeconfig={{ kubeconfig }} -n openshift-machine-api |grep {{ gpu_instance_type }} |awk '{ print $2 }'
```

Comment:
change `{{ gpu_instance_type` to `'{{ gpu_instance_type }}'` to avoid possible misuse/mistake